openmc/mgxs/mdgxs.py

```diff
@@ -1574,3 +1577,155 @@ def xs_tally(self):
             super(Beta, self)._compute_xs()
 
         return self._xs_tally
+
+
+class DecayRate(MDGXS):
+    r"""The decay rate for delayed neutron precursors.
+
+    This class can be used for both OpenMC input generation and tally data
+    post-processing to compute spatially-homogenized and energy-integrated
+    multi-group and multi-delayed group cross sections for multi-group
+    neutronics calculations. At a minimum, one needs to set the
+    :attr:`DecayRate.energy_groups` and :attr:`DecayRate.domain` properties.
+    Tallies for the flux and appropriate reaction rates over the specified
+    domain are generated automatically via the :attr:`DecayRate.tallies`
+    property, which can then be appended to a :class:`openmc.Tallies` instance.
+
+    For post-processing, the :meth:`MGXS.load_from_statepoint` will pull in the
+    necessary data to compute multi-group cross sections from a
+    :class:`openmc.StatePoint` instance. The derived multi-group cross section
+    can then be obtained from the :attr:`DecayRate.xs_tally` property.
+
+    For a spatial domain :math:`V`, energy group :math:`[E_g,E_{g-1}]`, and
+    delayed group :math:`d`, the decay rate is calculated as:
+
+    .. math::
+
+       \langle \lambda_d \nu^d \sigma_f \phi \rangle &= \int_{r \in V} dr
+       \int_{4\pi} d\Omega' \int_0^\infty dE' \int_0^\infty dE \; \chi(E) \nu^d
```

Comment:
Also, why does chi appear in these equations if it's just immediately integrated out?